src/desktop/components/main_layout/footer/upper.jade

```diff
@@ -12,6 +12,7 @@
   .mlf-upper-column
     h4 About Artsy
     a( href='/about' ) About
+    a( href='https://medium.com/artsy-blog' ) Blog
     a( href='/about/jobs' ) Jobs
     a( href='http://artsy.github.com/open-source/', target='_blank' ) Open Source
```

Comment:
Unrelated: Surprised this one is not `https://`

Reply:
Oh yeah good point. Maybe this link predates https by default on Medium. Anyway I'll change it 👍

Reply:
Whoops deleted earlier comment where I misread the link you were talking about — will fix this one in any case.